<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QGroupBox" name="taskGroup">
       <property name="title">
        <string>Task Settings</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="labelAutoDelete">
          <property name="text">
           <string>Auto-delete completed tasks:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="autoDeleteCheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDefaultDeadline">
          <property name="text">
           <string>Default deadline for Quick Add:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="defaultDeadlineBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelDeletePeriod">
          <property name="text">
           <string>Deletion period:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="deletePeriodBox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QGroupBox" name="appearanceGroup">
       <property name="title">
        <string>Appearance</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelTheme">
          <property name="text">
           <string>Theme:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="themeBox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QGroupBox" name="reminderGroup">
       <property name="title">
        <string>Reminders</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="1">
         <widget class="QCheckBox" name="reminderCheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelEnableRem">
          <property name="text">
           <string>Enable reminder:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelReminderTime">
          <property name="text">
           <string>Reminder time:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="reminderTimeBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1440</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelReminderSound">
          <property name="text">
           <string>Reminder sound:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="selectSoundButton">
          <property name="text">
           <string>Select sound...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelSoundPathLabel">
          <property name="text">
           <string>Selected file:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelSoundPath">
          <property name="text">
           <string>(none selected)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
